storage/tables/index.md

```diff
@@ -152,12 +152,53 @@ the result table will contain:
 |John|15|$150|
 |Darla|60|$600000|
 |John|34|$340000|
-|Annie|30|$500|
+|Annie|30|$500000|
 
-When importing data into a table with a primary key, the uniqueness is checked. 
+When importing data into a table with a primary key, the uniqueness is checked.
 The record `John,34,$340000` will overwrite the row `John,34,$340`, because it has the same primary key.
-The above applies only when **incremental load** is used. 
+The above applies only when **incremental load** is used.
+
+If an imported record does match an existing row but does not contain any change in values
+(eg. importing `John,34,$340`) the record will not be updated.
 
-When incremental is not used, the contents of the target table are cleared before the load. When a primary key 
+When incremental is not used, the contents of the target table are cleared before the load. When a primary key
 is not defined and an incremental load is used, it simply appends the data to the table and does not update anything.
 
+### Incremental Processing
+When a table is loaded incrementally the update time of each row is recorded internally. This information
+can be later used in Input Mapping of many components (especially [Transformations](/manipulation/transformations/mappings/#input-mapping)).
+In input mapping, you may specify the `Changed in last` option which allows the component to process only an increment of the data:
+
+{: .image-popup}
+![Screenshot - Incremental Processing](/storage/tables/incremental-processing.png)
+
+Extending the [above example](#incremental-loading) - if you have a table with a primary key defined on the columns `name` and `age`:
+
+|name|age|money|
+|---|---|---|
+|John|15|$150|
+|John|34|$340|
+|Darla|60|$600|
+
+and you import the following data to the table:
+
+|name|age|money|
+|---|---|---|
+|Annie|30|$500000|
+|John|34|$340000|
+|Darla|60|$600000|
+
+assuming that the import was on 2.1.2010 10:00 the result table will contain (the *updated* column is not an actual column of the table):
+
+|name|age|money|*updated*
+|---|---|---|---|
+|John|15|$150|1.1.2020 20:10|

```

Comment:
tady asi taky nějakou smysluplnější hodnotu

Reply:
a hlavně John byl teda taky updatovanej

Reply:
ne, tenhle vlastně ne! mně to zmátlo, co kdyby tam v tuhle chvíli nebyl složenej PK ale jenom jednoduchej?

Reply:
tak tam nepribyde john 34, meli bysme to jeste rozdelit na jednoduchy a slozeny pk?

Reply:
tuhle stránku jsem pochopil, že má bejt pro dummies, tak bych zmínil, že primární klíč může být i složenej, ale demonstroval to na jednosloupcovým. stalo se mi, že jsem musel v hlavě pracovat s 2 konceptama naráz - checkování složenýho PK a checkování změny hodnot. trochu mentální overhead.

Reply:
Should we maybe use the actual `_timestamp` name for the `*updated*` column?

Reply:
@ondrejhlavacek ok

Reply:
@pivnicek so far it's still hidden and inaccessible to the end-user

Reply:
yes, true, so I think it's ok then,  the `*` make it clear that it's not a "real" column name 👍